model/src/main/proto/exploration.proto

```diff
@@ -246,9 +246,19 @@ message UserAnswer {
     string plain_answer = 3;
     // A list of HTML textual representation of the answer.
     ListOfSetsOfHtmlStrings list_of_html_answers = 4;
+    // Contains a plain answer with an accessible version of the same answer.
+    AccessibleAnswer accessible_answer = 5;
```

Comment:
As discussed, suggest having a content description string property stored in UserAnswer so that any answer can have a content description specified.

Reply:
updated